animated-transformer/src/lib/tokens/token_gemb.ts

```diff
@@ -57,23 +57,38 @@ export function embed(
   return embeddedInput;
 }
 
-// TODO: consider supporting padding string[][] ?
-// pad(inputs: string[][], config: {
-//   paddingId: number;
-//   padAt: 'start' | 'end';
-//   dtype: tf.NumericDataType,
-// }) {
+// Maps tokens in string format to indexes.
+export function mapToIdx(
+  tokenToIdx: { [token: string]: number },
+  examples: string[][]
+): number[][] {
+  const tokenIdxs: number[][] = [];
+  examples.forEach((example) => {
```

Comment:
I think this is a bit simpler and clearer as: 

`return examples.map((example) => example.map((s) => tokenToIdx[s]))`

Reply:
Done.